### ISSUE TITLE
Can't install shotcut

### ISSUE THREAD

**Issue by tonejito** [Homebrew/homebrew-cask#186969]
### Verification

- [X] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [X] I have retried my command with `--force`.
- [X] I ran `brew update-reset && brew update` and retried my command.
- [X] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [X] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [X] I made doubly sure this is not a [checksum does not match / SHA256 mismatch](https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match) error (do not open an issue before trying to open a PR to fix first).

--------

### Description of issue

The `shotcut` software has an issue on the `24.09.13` release and the DMG binary does not have that version number:

- https://github.com/mltframework/shotcut/releases/tag/v24.09.13

The DMG binary should be `shotcut-macos-240913.dmg`, but it was overwritten as `shotcut-macos-240919.dmg` (`240913` was overwritten to `240919`)

The `24.09.19` version is currently marked as pre-release, so the checks fail because it is not "stable".

- https://github.com/mltframework/shotcut/releases/tag/v24.09.19

This issue has been reported to the upstream repository, but the maintainer stated that it might get fixed in homebrew, rather than upstream.

- https://github.com/mltframework/shotcut/issues/1583

--------

```
% brew install shortcut
...
curl: (56) The requested URL returned error: 404                                                                                                                                   

Error: Download failed on Cask 'shotcut' with message: Download failed: https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
```

--------

```
% brew install shotcut --force
==> Downloading https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
curl: (56) The requested URL returned error: 404                                                                                                                                   

Error: Download failed on Cask 'shotcut' with message: Download failed: https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
```

--------

```
% brew update-reset && brew update
...

% 
(.venv) tonejito@VF-0 ~ % brew install shotcut        
==> Downloading https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
curl: (56) The requested URL returned error: 404                                                                                                                                   

Error: Download failed on Cask 'shotcut' with message: Download failed: https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
```

--------

```
 % brew doctor
Your system is ready to brew.
(.venv) tonejito@VF-0 ~ % brew install shotcut
==> Downloading https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
curl: (56) The requested URL returned error: 404                                                                                                                                   

Error: Download failed on Cask 'shotcut' with message: Download failed: https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
```

### Command that failed

brew install shotcut

### Output of command with `--verbose --debug`

```shell
% brew install shotcut --verbose --debug
==> Downloading https://formulae.brew.sh/api/formula.jws.json
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --fail --remote-time --output /Users/tonejito/Library/Caches/Homebrew/api/formula.jws.json --location --time-cond /Users/tonejito/Library/Caches/Homebrew/api/formula.jws.json --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --fail --compressed --speed-limit 100 --speed-time 5 https://formulae.brew.sh/api/formula.jws.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 4448k  100 4448k    0     0  5808k      0 --:--:-- --:--:-- --:--:-- 5814k
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::NullLoader): loading shotcut
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::NullLoader): loading shotcut
==> Downloading https://formulae.brew.sh/api/cask.jws.json
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --fail --remote-time --output /Users/tonejito/Library/Caches/Homebrew/api/cask.jws.json --location --time-cond /Users/tonejito/Library/Caches/Homebrew/api/cask.jws.json --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --fail --compressed --speed-limit 100 --speed-time 5 https://formulae.brew.sh/api/cask.jws.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 1496k  100 1496k    0     0  4690k      0 --:--:-- --:--:-- --:--:-- 4691k
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::FromAPILoader): loading shotcut
==> Cask::Installer#install
==> Printing caveats
==> Cask::Installer#fetch
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --retry 3 --fail --location --silent --head https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --retry 3 -V
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --retry 3 --fail --location --silent --head --request GET https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --retry 3 -V
==> Downloading https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --retry 3 --fail --location --silent --head https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --retry 3 -V
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --retry 3 --fail --location --silent --head --request GET https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --retry 3 -V
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --fail --retry 3 --remote-time --output /Users/tonejito/Library/Caches/Homebrew/downloads/766846feb9a2621a7f3078e4f04a54a11d26dc90815d4e3f0516211dbd5d4cc9--shotcut-macos-240913.dmg.incomplete --location https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     9    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (56) The requested URL returned error: 404
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/4.3.24-128-ga27e09c\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 14.6.1\)\ curl/8.7.1 --header Accept-Language:\ en --retry 3 -V
Error: Download failed on Cask 'shotcut' with message: Download failed: https://github.com/mltframework/shotcut/releases/download/v24.09.13/shotcut-macos-240913.dmg
/usr/local/Homebrew/Library/Homebrew/download_strategy.rb:427:in `rescue in fetch'
/usr/local/Homebrew/Library/Homebrew/download_strategy.rb:424:in `fetch'
/usr/local/Homebrew/Library/Homebrew/downloadable.rb:102:in `fetch'
/usr/local/Homebrew/Library/Homebrew/cask/download.rb:60:in `fetch'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:193:in `download'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:74:in `fetch'
/usr/local/Homebrew/Library/Homebrew/cask/installer.rb:104:in `install'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:245:in `block in run'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:234:in `each'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:234:in `run'
/usr/local/Homebrew/Library/Homebrew/brew.rb:94:in `<main>'
```

### Output of `brew doctor` and `brew config`

```shell
% brew doctor             
Your system is ready to brew.

% brew config
HOMEBREW_VERSION: 4.3.24-128-ga27e09c
ORIGIN: https://github.com/Homebrew/brew
HEAD: a27e09cff41167471d0d75be3a56febb7e05b4c4
Last commit: 11 hours ago
Core tap JSON: 30 Sep 03:35 UTC
Core cask tap JSON: 30 Sep 03:35 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.5 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.5/bin/ruby
CPU: octa-core 64-bit icelake
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 14.6.1-x86_64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
```

### Output of `brew tap`

```shell
% brew tap

(no output)
```

**Comment by tonejito** [comment]
There were efforts to manually patch the Cask file to allow for this situation to be handled. However, the PRs were closed for a variety of reasons and the CI checks were not passing.

- #186672
- #186553
- #186487

I created a PR to revert the changes created on #185387 and fall back to the last version that passed the CI checks.

- 186968

This might not be the ideal solution, but we can create another PR to update to the next stable branch where the version numbers match after things have settled.

- https://github.com/mltframework/shotcut/releases
